pkg/metadata/metadata_store.go

```diff
@@ -0,0 +1,45 @@
+package metadata
+
+import (
+	"sync"
+)
+
+const (
+	DaprRuntimeVersionKey = "daprRuntimeVersion"
+)
+
+// Store is an interface to perform operations on metadata store.
+type Store interface {
+	MetadataGet() (map[string]string, error)
+	MetadataSet(key, value string) error
+}
+
+// DefaultStore is a default implementation of Store.
+type DefaultStore struct {
```

Comment:
What is the advantage of creating a new type?

Reply:
It is mainly used for runtime metadata storage, and is only stored in memory. In the future, runtime metadata may also be stored in other storage components.